### ISSUE TITLE
Powershell errors out on Win2008R2

### ISSUE THREAD

**Issue by DBa2016** [PowerShell/PowerShell#18971]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://github.com/PowerShell/PowerShell/releases)
- [X] Search the existing issues.
- [X] Refer to the [FAQ](https://github.com/PowerShell/PowerShell/blob/master/docs/FAQ.md).
- [X] Refer to [Differences between Windows PowerShell 5.1 and PowerShell](https://docs.microsoft.com/powershell/scripting/whats-new/differences-from-windows-powershell).

### Steps to reproduce

Installed the MSI package on Win2008R2 and started pwsh. Tried both 7.3.1 and 7.4.0-Preview.

### Expected behavior

```console
Starting PWSH without an exception.
```

### Actual behavior

```console
Immediately receiving an exception:
`
Unhandled exception. System.TypeInitializationException: The type initializer for 'System.Management.Automation.Tracing.PSEtwLog' threw an exception.
 ---> System.TypeInitializationException: The type initializer for 'System.Management.Automation.Tracing.PSEtwLogProvider' threw an exception.
 ---> System.DllNotFoundException: Unable to load DLL 'api-ms-win-eventing-provider-l1-1-0.dll' or one of its dependencies: The specified module could not be found. (0x8007007E)
   at System.Diagnostics.Eventing.UnsafeNativeMethods.EventRegister(Guid& providerId, EtwEnableCallback enableCallback, Void* callbackContext, Int64& registrationHandle)
   at System.Diagnostics.Eventing.EventProvider.EtwRegister()
   at System.Management.Automation.Tracing.PSEtwLogProvider..cctor()
   --- End of inner exception stack trace ---
   at System.Management.Automation.Tracing.PSEtwLog..cctor()
   --- End of inner exception stack trace ---
   at System.Management.Automation.Tracing.PSEtwLog.LogConsoleStartup()
   at Microsoft.PowerShell.UnmanagedPSEntry.Start(String[] args, Int32 argc)
   at Microsoft.PowerShell.ManagedPSEntry.Main(String[] args)
````
```


### Error details

```console
n/a
```

### Environment data

```powershell
n/a
```

**Comment by mklement0** [comment]
Per https://learn.microsoft.com/en-us/powershell/scripting/install/PowerShell-Support-Lifecycle?view=powershell-7.3#supported-platforms, the minimum server version supported is _Windows Server 2012 R2_.

If there's an official installation mechanism that didn't detect the incompatibility _at installation time_, that would be problematic; possibly related:

* https://github.com/PowerShell/PowerShell/issues/18909

**Comment by DBa2016** [comment]
There was no warning and no error message. Tried double-clicking the MSI file as well as running msiexec with some "batch" arguments - both cleanly installed, and both refused to run.

Is there any powershell 7 version working on Win2k8R2?

**Comment by mklement0** [comment]
@DBa2016, that is unfortunate, but it is a separate problem that must be addressed in the installer, as discussed in  #18909

> Is there any powershell 7 version working on Win2k8R2?

The article first linked to in my previous comment lists the supported platforms for the still-supported PowerShell (Core) versions.

That comment also summarizes the linked article by stating that the minimum version listed there is  _Windows Server 2012 R2_. 

I think it is safe to conclude that _Windows Server 2008 R2_ is _not_ supported (unless someone knows of a hack).

**Comment by Temik007** [comment]
Solution - https://github.com/PowerShell/PowerShell/issues/19816
